src/bin/stratis_uuids_to_names.rs

```diff
@@ -0,0 +1,201 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::{
+    collections::HashMap,
+    env::args,
+    error::Error,
+    fmt::{self, Debug, Display},
+    os::unix::net::UnixDatagram,
+    time::Duration,
+};
+
+use dbus::{
+    arg::{RefArg, Variant},
+    blocking::Connection,
+    Path,
+};
+use lazy_static::lazy_static;
+use regex::Regex;
+use uuid::Uuid;
+
+pub const DEV_LOG: &str = "/dev/log";
+pub const SYSTEM_DAEMON_INFO: &str = "<30>";
```

Comment:
I couldn't figure out where these two numbers come from or why they're in brackets.